### ISSUE TITLE
Reference numbering

### ISSUE THREAD

**Issue by matthewjay15** [UCL-ECHILD/docs.echild.ac.uk-website#50]
I've noticed the reference numbering system may be prone to creating confusion. The refs are numbered sequentially from 1 on each page but these numbers do not correspond to the numbers in the reference list, which would be confusing for academic readers (it confused me just now). Also, when clicking a reference hyperlink, one is directed to the references page with the selected reference at the top, however, on my browser at least, the actual reference is hidden by the browser toolbar.

Is it possible to alter the numbering system so the in-text numbers match the reference list numbers?

**Comment by tony-stone** [comment]
@matthewjay15, yes, I noticed this also. I think this is a base quarto behaviour and likely not to be modifiable (without considerable effort).

One option would be to use a different referencing style specification, e.g. using of first author's surname and test of publication - as per Harvard style. This just requires replacing the csl file in the web_source directory and  specified in 'web_source/_quarto.yml'

See also:
https://quarto.org/docs/authoring/footnotes-and-citations.html

**Comment by matthewjay15** [comment]
Shall we give the Harvard style a go, then? Looks easy enough.

**Comment by matthewjay15** [comment]
Going to close this now. The multiple primary authors per year is fine imo as other authors are displayed.
